ebos/eclwriter.hh

```diff
@@ -603,8 +593,11 @@ private:
 
         // Cartesian index mapper for the serial I/O grid
         const auto& equilCartMapper =  simulator_.vanguard().equilCartesianIndexMapper();
-
         const auto& cartDims = simulator_.vanguard().cartesianIndexMapper().cartesianDimensions();
+        auto nncCompare =  []( const Opm::NNCdata& nnc1, const Opm::NNCdata& nnc2){
+                               return nnc1.cell1 < nnc2.cell1 ||
+                                      ( nnc1.cell1 == nnc2.cell1 && nnc1.cell2 < nnc2.cell2);};
```

Comment:
Since we're moving this around anyway, maybe we could switch to implementing `operator<` in terms of `tuple/tie`?
```C++
auto nncCompare = [](const Opm::NNCData nnc1, const Opm::NNCData& nnc2)
{
    return std::tie(nnc1.cell1, nnc1.cell2) < std::tie(nnc2.cell1, nnc2.cell2);
};
```

Reply:
I thought of that - the reason I did not do it was that I was a bit uncertain about the expected semantics from the `trans` member.